lib/components/src/placeholder/placeholder.tsx

```diff
@@ -15,7 +14,11 @@ const Message = styled.div`
   font-size: ${props => props.theme.typography.size.s2 - 1}px;
 `;
 
-export const Placeholder = ({ children, ...props }) => {
+// Need to define Placeholder using old fashioned `function Placeholder` to avoid TS compiler to
```

Comment:
Is this still valid after the change to use babel for compilation?

Reply:
I will revert my commit to try ;)

Reply:
🎉